cms/templates/settings.html

```diff
@@ -306,6 +306,21 @@ <h2 class="title-2">${_("Requirements")}</h2>
                   <input type="text" class="short time" id="course-effort" placeholder="HH:MM" />
                   <span class="tip tip-inline">${_("Time spent on all course work")}</span>
                 </li>
+                % if settings.FEATURES.get('ENABLE_PREREQUISITE_COURSES') and settings.FEATURES.get('MILESTONES_APP'):
```

Comment:
What is the workflow for setting this information for courses on edX (where about_page_editable returns False)?